### PR TITLE
Remove left check from safe is funded

### LINKED ISSUE #246: Step 2-1: Safe is funded
Remove left checmark from svg src: '/assets/safe-success-2.svg',
Update copy on the slide

### PR DESCRIPTION
**Pull request by diegoalzate**
closes #246 

### overview
remove left check mark from svg shown in `safe is ready` and updated copy.

### CODE REVIEW

File: src/pages/staking-hub/onboarding/step0/1createSafe.tsx [hunk 1]
```diff
@@ -117,23 +117,25 @@ const CreateSafe = () => {
...
+                  onChange={(e) => updateOwnerAddress(owner.id, e.target.value)}
+                />
+                <IconButton onClick={() => removeOwner(owner.id)}>
+                  <DeleteIcon />
+                </IconButton>
+              </FlexContainer>
+            );
+          })
+          : null}
+      </div>
```
Comment (mjadach-iv) [inline src/pages/staking-hub/onboarding/step0/1createSafe.tsx:138]: I do not see the reason why you would change the `a && b` into  `a ? b  :  null` here and leave the  `a && b`  logic below?

Comment (diegoalzate) [reply src/pages/staking-hub/onboarding/step0/1createSafe.tsx:138]: yeah this could stay how it was

Comment (mjadach-iv) [reply src/pages/staking-hub/onboarding/step0/1createSafe.tsx:138]: while on it, you can also bring back `{(is_xDAI_to_safe_loading || is_wxHOPR_to_safe_loading) && <span>Check your Wallet...</span>}`

Comment (diegoalzate) [reply src/pages/staking-hub/onboarding/step0/1createSafe.tsx]: brought all of that back

File: public/assets/safe-success-2.svg
Comment (mjadach-iv) [inline public/assets/safe-success-2.svg:1]: You only need to remove the left check, without moving the rest around. The safe needs to be centered after the removal

**Review by mjadach-iv — CHANGES_REQUESTED** [review]

**Pull request closed without merging**